src/configs/fennel.rs

```diff
@@ -13,7 +13,7 @@ pub struct FennelConfig<'a> {
     pub symbol: &'a str,
     pub style: &'a str,
     pub disabled: bool,
-    pub detect_extentions: Vec<&'a str>,
+    pub detect_extensions: Vec<&'a str>,
```

Comment:
I would prefer not to break compatibility with older configs with this change:
```suggestion
    #[serde(alias = "detect_extentions")]
    pub detect_extensions: Vec<&'a str>,
```

Reply:
How about keeping the old config and adding a TODO?

Reply:
PTAL

Reply:
We have some alias for similar cases, I would prefer using that here; but please add a TODO, to ensure it will be remembered on the next breaking release.

Reply:
Done. PTAL

Btw, this typo only happens in fennel.